django_project/feti/api_views/occupation.py

```diff
@@ -0,0 +1,21 @@
+from rest_framework.response import Response
+from rest_framework.views import APIView
+
+from feti.models.occupation import Occupation
+from feti.serializers.occupation_serializer import OccupationListSerializer
+
+__author__ = 'Alison Mukoma <alison@kartoza.com>'
+__date__ = '24/10/17'
+__license__ = "GPL"
+__copyright__ = 'kartoza.com'
+
+
+class OccupationListApiView(APIView):
+    """
+    Api to list all available occupations.
+    """
+    def get(self, request):
+        occupations = Occupation.objects.order_by('occupation')
```

Comment:
Maybe better write it Occupation.objects.all().order_by('occupation')

Reply:
Great,I have updated @ann26